x-pack/plugins/lens/readme.md

```diff
@@ -12,3 +12,31 @@ Run all tests from the `x-pack` root directory
 - API Functional tests:
   - Run `node scripts/functional_tests_server`
   - Run `node ../scripts/functional_test_runner.js --config ./test/api_integration/config.ts --grep=Lens`
+
+
+## UI Terminology
+
+Lens has a lot of UI elements – to make it easier to refer to them in issues or bugs, this is a hopefully complete list:
+
+* **Top nav** Navigation menu on top of the app (contains Save button)
+  * **Query bar** Input to enter KQL or Lucene query below the top nav
+  * **Filter bar** Row of filter pills below the query bar
+  * **Time picker** Global time range configurator right to the query bar
+* **Data panel** Panel to the left showing the field list
+  * **Field list** List of fields separated by available and empty fields in the data panel
+  * **Index pattern chooser** Select element switching between index patterns
+  * **Field filter** Search and dropdown to filter down the field list
+  * **Field information popover** Popover showing data distribution; opening when clicking a field in the field list
+* **Config panel** Panel to the right showing configuration of the current chart, separated by layers
```

Comment:
I call this the `editor`. Also, since Lens uses more than charts, can we use `visualizations` instead? 

So it would look something like:

**Editor** Specifies the visualization configuration options, separated by layers